main.nf

```diff
@@ -463,54 +529,97 @@ def SHOW_USEAGE() {
         if (helpFile.exists()) {
             println helpFile.text
         } else {
-            println "Sorry something went wrong, usage file not found! please vist our website for more info : https://ai-marrvel.readthedocs.io/en/latest/"
+            println """
+            Sorry something went wrong, usage file not found!
+            Please vist our website for more info : https://ai-marrvel.readthedocs.io/en/latest/
+            """
         }
         exit 0
     }
 }
 
+process VALIDATE_INPUT_PARAMS {
+    // Later will publish there validation error file into its own directory for webApp use.
+    input:
+    path vcf
+    path hpo
+    path refDir
+
+    output:
+    val isValidationPassed
+
+    script:
+    def userParamsJson = groovy.json.JsonOutput.toJson(params)
+    isValidationPassed = true
+    """
+    validate_inputs.py '${userParamsJson}' $vcf $hpo $refDir
+
+    file_path="\${PWD}/validation_results.csv"
+
+    if [ -f "\$file_path" ]; then
+        check_validation_status.py \$file_path
+    else
+        echo "CSV file not found: \$file_path"
+        exit 1
+    fi
+    """
+}
+
 workflow {
```

Comment:
please remove all the redundant VALIDATE_INPUT_PARAMS.out except for `BUILD_REFERENCE_INDEX` and `INDEX_VCF` after you changed the above lines.